circuitpy/lib/farm_ng/utils/can.py

```diff
@@ -13,7 +13,7 @@ def setup_can_default():
     """Sets up the default canio.CAN connection, for your board type Uses baud_rate of 250K (default bus speed for
     the Amiga)"""
     board_type = get_board_type()
-    baud_rate = 250_000
+    baud_rate = 1_000_000
```

Comment:
TODO this needs to be easily changed from client code.

Reply:
can.py:
```
def setup_can_default(baud_rate: int = 250_000):
    """Sets up the default canio.CAN connection, for your board type Uses baud_rate of 250K (default bus speed for
    the Amiga)"""
    board_type = get_board_type()
    if board_type == BoardType.AMIGA_DISPV0:
        ...
```
main_loop.py:
```
class MainLoop:
    """Main driver for all farm-ng apps run on microcontrollers."""

    def __init__(self, AppClass, has_display=True, baud_rate: int =  250_000) -> None:
        ...
        self.can = setup_can_default(baud_rate)
        ...
```

example/foo.py
```
...
def main():
    MainLoop(AppClass=RmdServoApp, has_display=False, baud_rate=1_000_000).loop()

main()
```